channeldb/invoices.go

```diff
@@ -154,8 +164,19 @@ type ContractTerm struct {
 	// which can be satisfied by the above preimage.
 	Value lnwire.MilliSatoshi
 
-	// State describes the state the invoice is in.
-	State ContractState
+	// FinalCltvDelta is the minimum required number of blocks before htlc
+	// expiry when the invoice is accepted.
+	FinalCltvDelta int32
+
+	// Expiry defines how long after creation this invoice should expire.
+	Expiry time.Duration
+
+	// PaymentAddr is a randomly generated value include in the MPP record
+	// by the sender to prevent probing of the receiver.
+	PaymentAddr [32]byte
```

Comment:
i'd rather add these fields during the migration so that users can't accidentally revert to a commit where we don't process them. the downgrade logic in opening channeldb will prevent this if they are added here

Reply:
Does this mean that we are going to increase the db version with every tlv field that we add, a no-op migration>

Reply:
i think it's the safest path

Reply:
Ok. It is easy to forget though. Should we add a NOTE comment to every list of db tlv type constants in the code base to remind future devs about this? Pre-existing, but maybe not worth the overhead of a separate pr

Reply:
done